src/rendition-mixin.js

```diff
@@ -87,7 +92,8 @@ let renditionSelectionMixin = function(hlsHandler) {
     return playlists
       .master
       .playlists
-      .map((e, i) => new Representation(hlsHandler, e, i));
+      .filter((media) => !isBlacklisted(media))
```

Comment:
This may get overly complicated, but would it be better for us to use something else to identify incompatible variants versus temporary suspensions? It may make more contextual sense, but if it ends up being too many options, the check for not Infinity could be an OK solution for now.